### DISCUSSION TITLE
Any way to get numbering as english words?

### DISCUSSION THREAD

**Discussion by A-Walrus** [typst/typst#3956] [Q&A] [answered]
Something like:
- Chapter One
- Chapter Two
...
- Chapter Fourteen

**Comment by EpicEricEE** [comment, marked as answer]
You could use the [`name-it`](https://github.com/RubixDev/typst-name-it) package like this:
```typ
#import "@preview/name-it:0.1.1": name-it

#show heading.where(level: 1): set heading(
  supplement: [Chapter],
  numbering: num => {
    let name = name-it(num)
    name = upper(name.at(0)) + name.slice(1)
    [#name]
  }
)

#show heading.where(level: 1): it => {
  let num = counter(heading).display(it.numbering)
  [#it.supplement #num --- #it.body]
}

= Something <first>

= Something else

See @first.
```

![image](https://github.com/typst/typst/assets/7191192/b61c304f-55b2-4781-930b-bffb129e6cfb)

It may look a bit weird in the outline though, as it only says "One Something" for the first chapter, so that will probably need some adjustment.
